pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -1946,16 +1946,15 @@ statfunc u32 cgroup_skb_generic(struct __sk_buff *ctx, enum flow_direction flow_
         //
         // NOTE: this will not work for nested cgroups, but it is an accepted limitation for now.
         if (!existing_netctx) {
-            net_task_context_t cgroup_context = {0};
-            cgroup_context.taskctx.cgroup_id = bpf_sk_cgroup_id(sk);
-            existing_netctx = &cgroup_context;
```

Comment:
This is the real bug, we are setting `existing_netctx` addr to a local variable in the `if` block (`&cgroup_context`).